packages/common/package.json

```diff
@@ -20,37 +20,10 @@
     "./utils": "./dist/utils.js",
     "./kms": "./dist/kms.js"
   },
-  "typesVersions": {
-    "*": {
-      "index": [
-        "./src/index.ts"
-      ],
-      "actions": [
-        "./src/actions/index.ts"
-      ],
-      "chains": [
-        "./src/chains/index.ts"
-      ],
-      "codegen": [
-        "./src/codegen/index.ts"
-      ],
-      "errors": [
-        "./src/errors/index.ts"
-      ],
-      "foundry": [
-        "./src/foundry/index.ts"
-      ],
-      "type-utils": [
-        "./src/type-utils/index.ts"
-      ],
-      "utils": [
-        "./src/utils/index.ts"
-      ],
-      "kms": [
-        "./src/exports/kms.ts"
-      ]
-    }
-  },
+  "main": "./dist/index.js",
```

Comment:
why did `main` get added here and not in others?

Reply:
moduleResolution "Node" requires this be specified here as IIRC it doesn't respect exports.

I believe only this package was being imported in a problematic way, but to be on the safe side it might be best as a followup to add this to the other packages as well for the primary entry point.